ot/da.py

```diff
@@ -1793,3 +1795,122 @@ def transform(self, Xs):
                 transp_Xs = K.dot(self.mapping_)
 
             return transp_Xs
+
+
+class UnbalancedSinkhornTransport(BaseTransport):
+
+    """Domain Adapatation unbalanced OT method based on sinkhorn algorithm
+
+    Parameters
+    ----------
+    reg_e : float, optional (default=1)
```

Comment:
Perhaps the default `reg_e` should be smaller ? or None and set to 10 / dimension for e.g in `fit`

Reply:
I agree, but since by default norm = None, maybe we could leave it like that to avoid numerical errors in the default run?

Reply:
Actually I just realized that I have to check if the norm is None and do no normalization in that case, otherwise the error message will be triggered...

Reply:
Ok it is done and it seems ok

Reply:
Ok I just realized the other `da` classes use the same default